src/guides/v2.4/graphql/mutations/generate-customer-token-as-admin.md

```diff
@@ -0,0 +1,64 @@
+---
+group: graphql
+title: generateCustomerTokenAsAdmin mutation
+b2b_only: true
+contributor_name: EY 
+---
+
+Use the `generateCustomerTokenAsAdmin` mutation to generate a new customer token as an admin so that an administrator can perform remote shopping assistance.
+The customer must have enabled the `allow_remote_shopping_assistance` feature while creating the customer profile.
+
+## Syntax
+
+```graphql
+mutation {generateCustomerTokenAsAdmin(input: GenerateCustomerTokenAsAdminInput!) {GenerateCustomerTokenAsAdminOutput}}
+```
+
+## Example usage
+
+The following call creates a new customer token.
+
+**Request:**
+
+```graphql
+mutation{
+  generateCustomerTokenAsAdmin(input: {
+    customer_email: "customer1@mail.com"
+  }){
+    customer_token
+  }
+}
+```
+
+**Response:**
+
+```json
+{
+  "data": {
+    "generateCustomerTokenAsAdmin": {
+      "customer_token": "cr0717abzoagxty1xjn4lj13kim36r6x"
+    }
+  }
+}
+```
+
+## Input attributes
+
+The `generateCustomerTokenAsAdmin` mutation requires the following input attribute:
+
+Attribute |  Data Type | Description
+--- | --- | ---
+`customer_email` | String | The customer's email address
```

Comment:
Please note, `customer_email` is a required attribute

```suggestion
`customer_email` | String! | The customer's email address
```

Reply:
Ah yes! My bad. Thanks. @rogyar